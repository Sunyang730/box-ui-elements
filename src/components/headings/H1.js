// @flow
import * as React from 'react';
import classNames from 'classnames';

import './H1.scss';

type Props = {
    children: React.Node,
    className?: string,
};

const H1 = ({ children, className }: Props) => <h1 className={classNames('bdl-H1', className)}>{children}</h1>;

export default H1;
