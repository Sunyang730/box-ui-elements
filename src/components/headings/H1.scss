@import '../../styles/variables';

h1.bdl-H1 {
    @include font-family;

    color: $bdl-gray;
    font-size: 22px;
    font-weight: bold;
    letter-spacing: .3px;
    line-height: 28px;
}
