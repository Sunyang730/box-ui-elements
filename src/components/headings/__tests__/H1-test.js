// @flow
import * as React from 'react';

import H1 from '../H1';

describe('components/headings/H1', () => {
    const getWrapper = (props = {}) => shallow(<H1 {...props}>foo</H1>);

    test('should render H1', () => {
        const wrapper = getWrapper({ className: 'moo' });

        expect(wrapper).toMatchSnapshot();
    });
});
