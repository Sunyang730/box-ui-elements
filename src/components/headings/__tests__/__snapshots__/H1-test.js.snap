// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/headings/H1 should render H1 1`] = `
<h1
  className="bdl-H1 moo"
>
  foo
</h1>
`;
